src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
+
+import environment from 'platform/utilities/environment';
+import { createIsServiceAvailableSelector } from 'platform/user/selectors';
+import backendServices from 'platform/user/profile/constants/backendServices';
+
+import ProfileFieldHeading from 'applications/personalization/profile360/vet360/components/base/ProfileFieldHeading';
+
+import PaymentInformationEditModal from '../components/PaymentInformationEditModal';
+import {
+  fetchPaymentInformation,
+  savePaymentInformation,
+  setPaymentInformationUiState,
+} from '../actions/paymentInformation';
+
+function isGated() {
+  return environment.isProduction();
+}
+
+class PaymentInformation extends React.Component {
+  static propTypes = {
+    isLoading: PropTypes.bool.isRequired,
+    isEligible: PropTypes.bool.isRequired,
+    fetchPaymentInformation: PropTypes.func.isRequired,
+    paymentInformation: PropTypes.shape({
+      responses: PropTypes.arrayOf(
+        PropTypes.shape({
+          paymentAccount: PropTypes.shape({
+            accountNumber: PropTypes.string.isRequired,
+            accountType: PropTypes.string.isRequired,
+            financialInstitutionName: PropTypes.string.isRequired,
+            financialInstitutionRoutingNumber: PropTypes.string.isRequired,
+          }),
+        }),
+      ),
+    }),
+  };
+
+  componentDidMount() {
+    if (!isGated() && this.props.isEligible) {
+      this.props.fetchPaymentInformation();
+    }
+  }
+
+  toggleEditModal = () => {
+    this.props.setPaymentInformationUiState({
+      isEditing: !this.props.paymentInformationUiState.isEditing,
+    });
+  };
+
+  render() {
+    if (isGated()) {
+      return null;
+    }
+
+    if (!this.props.isEligible) {
+      return null;
+    }
+
+    if (this.props.isLoading) {
+      return <LoadingIndicator message="Loading payment information..." />;
+    }
```

Comment:
There will be a couple other layers here in a follow-up PR - the "add" state for when the user is eligible for direct deposit but has not yet set it up, and we'll also be enforcing MFA to use this tool.

Reply:
MFA?

Reply:
Ah yeah I mean 2FA (text-message account verification during login configured in ID.me)